src/System.Linq.Expressions/tests/BinaryOperators/Comparison/EqualNotEqualTests.cs

```diff
@@ -28,6 +28,7 @@ public static IEnumerable<object[]> TestData()
                 yield return new object[] { new ushort[] { 0, 1, ushort.MaxValue }, useInterpreter };
                 yield return new object[] { new TestClass[] { new TestClass(), new TestClass() }, useInterpreter };
                 yield return new object[] { new TestEnum[] { new TestEnum(), new TestEnum() }, useInterpreter };
+                yield return new object[] { new E[] {E.A, E.B, (E)int.MinValue}, useInterpreter };
```

Comment:
This and the case below added in case the above had an impact. Should probably have been in anyway.